concrete/src/Cache/Adapter/ZendCacheDriver.php

```diff
@@ -109,20 +162,24 @@ protected function internalSetItem(&$normalizedKey, &$value)
      */
     protected function internalRemoveItem(&$normalizedKey)
     {
-        /** @var Cache $cache  */
-        $cache = Core::make($this->cacheName);
-        $item = $cache->getItem('zend/'.$normalizedKey);
+        $cache = $this->getCache();
+        if ($cache === null) {
+            $result = true;
```

Comment:
It depends on the meaning you give.

1. If `internalRemoveItem` means "please be sure that that item does not exist in the cache", we should return true.
2. If `internalRemoveItem` means "please remove that item and tell me if you succeeded", we should return false.

I adopted 1, but also 2 makes sense... I'll update my code.